sklearnex/tests/test_monkeypatch.py

```diff
@@ -0,0 +1,66 @@
+#===============================================================================
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+import sklearnex
+from daal4py.sklearn import sklearn_patch_map

```

Comment:
daal4py?

Reply:
this function is not available in sklearnex

Reply:
get_patch_names ? No?

Reply:
get_patch_names gets only **values**, but we need **keys** in this file